src/utilities.js

```diff
@@ -56,13 +56,6 @@ export function addOnActivateButtonEvent ($element,config,layout,url,recipient,t
       textboxReference.addEventListener('click', function() {
         var copyTextarea = document.querySelector('.linkTextboxArea');
         copyTextarea.select();
-        try {
-          var successful = document.execCommand('copy');
```

Comment:
`document.execCommand('copy');` is what's actually copying to clipboard, pretty sure you need to leave it in there.

Reply:
aha
i thought it only was for the sake of logging a successful/unsuccessful message